docs/overview.rst

```diff
@@ -157,8 +157,8 @@ testing. For the ocean core, they are here:
 
 .. code-block:: bash
 
-    $ ls testing_and_setup/compass/ocean/regression_suites/
-    land_ice_fluxes.xml  light.xml  nightly.xml  rpe_tests.xml
+    ls testing_and_setup/compass/ocean/regression_suites/
+       land_ice_fluxes.xml  light.xml  nightly.xml  rpe_tests.xml
```

Comment:
@mark-petersen, I don't agree with this change.  The `$` is a standard notation for differentiating something typed at the command line from the output it produces.

Reply:
But if we have $ everywhere, we can't copy a block of text. That is quite inconvenient. I know what you mean about differentiating between terminal input and output.

Reply:
I wouldn't put it everywhere, just places where we have both a command and its output.  This is only done in places where one would be copying a single command anyway (because you don't want to copy the output).  It's important to have a clear distinction between command and output and I don't think your indentation does this.